File: src/main/java/no/nav/melosys/eessi/models/buc/Action.java
package no.nav.melosys.eessi.models.buc;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import lombok.Data;

@Data
@JsonIgnoreProperties(ignoreUnknown = true)
public class Action {
    private String name;
    private String documentType;
}

File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/models/buc/Action.kt
package no.nav.melosys.eessi.models.buc;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@JsonIgnoreProperties(ignoreUnknown = true)
public class Action {
    private String name;
    private String documentType;
    private String documentId;
    private String operation;
}
